src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -398,13 +402,31 @@ enum class InterfaceConfigType : uint8_t {
   /**
    * Enable/disable the interface.
    *
-   * Valid for:
-   * - @ref TransportType::TCP_CLIENT
-   * - @ref TransportType::UDP_CLIENT
+   * Valid for all @ref TransportType%s.
```

Comment:
`%` = `'` ?

Reply:
No, this is doxy magic so you get a link to the type with an s after it like `TransportType`s. If you don't add the % you'll get a doxy warning that it doesn't know the class `TransportTypes`. I guess I could just change this to `all @ref TransportType values` maybe to avoid confusion.